mappings/net/minecraft/client/options/MainHandOption.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS aie net/minecraft/client/options/MainHandOption
```

Comment:
Git's not detecting it as a conflict, but this has been mapped to `net/minecraft/util/AbsoluteHand`

Reply:
Resolved, though it's not fair that I have to resolve conflicts while I've done the pr before. But whatever ^^